tests/test_users/test_routers.py

```diff
@@ -103,7 +103,7 @@ def test_anonymous_user_cannot_update_user(
     sync_client: TestClient,
     update_data: dict[str, str]
 ):
-    response = sync_client.put("users/1", json=update_data)
+    response = sync_client.patch("users/1", json=update_data)

```

Comment:
Nie tylko anonimowy user nie powinien moc update'owac kogos, nikt nie powinien miec uprawnien zeby czyjes dane zmienaic

Reply:
upgrade uwierzytelniania i autoryzacji spoko ale to w innym PR